docs/source/monitors.rst

```diff
@@ -144,8 +146,69 @@ Is there a **Basic Scrapy Suite** ready to use?
 Of course, there is! We really want to make it easy for you to monitor your spiders ;)
 
 .. automodule:: spidermon.contrib.scrapy.monitors
-    :members: SpiderCloseMonitorSuite 
+    :members: SpiderCloseMonitorSuite
 
+Periodic Monitors
+-----------------
+
+Sometimes we don't want to wait until the end of the spider execution to monitor
+it. For example, we may want to be notified as soon the number of errors reach
+a value or close the spider if the time elapsed is greater than expected.
+
+You define your `Monitors`_ and `Monitor Suites`_ the same way as before, but
+you need to provide the time interval (in seconds) between each of the times the
+`Monitor Suites`_ is run.
+
+In the following exemple, we defined a periodic monitor suite that will be
+executed every minute and will verify if the number of errors found is lesser
+than a value. If not, the spider will be closed.
+
+First we define a new action that will close the spider when executed:
+
+.. code-block:: python
+
+    # myproject/actions.py
+    from spidermon.core.actions import Action
+
+    class CloseSpiderAction(Action):
+
+        def run_action(self):
+            spider = self.data['spider']
+            spider.logger.info("Closing spider")
+            spider.crawler.engine.close_spider(spider, 'closed_by_spidermon')
+
+Then we create our monitor and monitor suite that verifies the number of errors
+and then take an action if it fails:
+
+.. code-block:: python
+
+    # myproject/monitors.py
+    from myproject.actions import CloseSpiderAction
+
+    @monitors.name('Periodic job stats monitor')
+    class PeriodicJobStatsMonitor(Monitor, StatsMonitorMixin):
+
+        @monitors.name('Maximum number of errors exceeded')
```

Comment:
```suggestion
        @monitors.name('Maximum number of errors reached')
```

It seems a bit more accurate, but feel free to ignore this comment.

If you do decide to apply it, please consider applying it to all the new code, as there are a few more cases.